### PR TITLE
Add `previous name` synonym type

### LINKED ISSUE #139: Request for adding property `Previous Name` to OMO
### IRI

_No response_

### Label

Previous Name

### Definition of the property

This is an additional synonym type that is used for synonyms that used to be the primary label. Several resources use this, notably HGNC

### Parent property

http://www.geneontology.org/formats/oboInOwl#SynonymType

### What is the range of the property in question?

xsd:string

### Examples of use

MAPT, the gene that encodes the Tau protein, has a previous name DDPAC. Note: in this case, the name type is more specifically the gene symbol

### Motivation to add

Following #124 and #133, this is another useful synonym type that can be used to help standardize hash urls used throughout OBO Foundry

### ORCID, ROR or Wikidata identifier of the contributor

https://orcid.org/0000-0003-4423-4370

### OMO Checklist

- [X] I believe the property is generally useful beyond my specific ontology needs.
- [X] There is no other property in OMO that covers the same use case.

### PR DESCRIPTION
**Pull request by cthoyt**
Closes #139

### CODE REVIEW

File: src/templates/annotation_properties.tsv [hunk 1]
```diff
@@ -8,3 +8,4 @@ OMO:0003004	owl:AnnotationProperty	plural form	A synonym type for describing plu
...
 OMO:0003006	owl:AnnotationProperty	misspelling	A synonym type for common misspellings	oboInOwl:SynonymType	https://orcid.org/0000-0003-4423-4370	2023-03-03		https://github.com/information-artifact-ontology/ontology-metadata/issues/132
 OMO:0003007	owl:AnnotationProperty	misnomer	A synonym type for misnomers, i.e., a synonym that is not technically correct but is commonly used anyway	oboInOwl:SynonymType	https://orcid.org/0000-0003-4423-4370	2023-03-03		https://github.com/information-artifact-ontology/ontology-metadata/issues/133
+OMO:0003008	owl:AnnotationProperty	previous name	A synonym type for previous names that used to be the primary label. names	oboInOwl:SynonymType	https://orcid.org/0000-0003-4423-4370	2023-07-25	MAPT, the gene that encodes the Tau protein, has a previous name DDPAC. Note: in this case, the name type is more specifically the gene symbol.	https://github.com/information-artifact-ontology/ontology-metadata/issues/139
```
Comment (matentzn) [inline src/templates/annotation_properties.tsv:11]:
```suggestion
OMO:0003008	owl:AnnotationProperty	previous name	A synonym type for names that have been used as primary labels in the past.	oboInOwl:SynonymType	https://orcid.org/0000-0003-4423-4370	2023-07-25	MAPT, the gene that encodes the Tau protein, has a previous name DDPAC. Note: in this case, the name type is more specifically the gene symbol.	https://github.com/information-artifact-ontology/ontology-metadata/issues/139

```

**Review by matentzn — COMMENTED** [review]
One suggestion, then we can release it.

**Comment by cthoyt** [comment]
Done

**Review by matentzn — APPROVED** [review]
Thank your as always for your contributions @cthoyt!

**Pull request merged**